docs/developer_guide.rst

```diff
@@ -35,3 +35,17 @@ Then, all that is needed for this to be automatically included when you run the
 the modules flag ``-m`` or ``--modules`` along with the name of your module that contains the custom check. If using
 the library instead, you need only import the ``available_checks`` global variable from your own submodules, or
 otherwise import your check functions after importing the ``nwbinspector`` in your ``__init__.py``.
+
+
+Disable Tests That Require Network Connection
+---------------------------------------------
+
+Some of the tests in the suite require internet connectivity both to and from the DANDI archive S3 bucket.
+If this is failing for some reason, you can explicitly control all related tests by setting the environment variable
+``DANDI_TESTS_NONETWORK`` to some value able to be parsed by ``distutils.util.str2tool``. For example, to disable them on
+a linux system, run
+
+.. code-block::
+    export DANDI_TESTS_NONETWORK=1
```

Comment:
I think something along the lines ``SKIP_NETWORK_TESTS`` would be more intuitive and also make it clear that this should apply to all tests that require network access. To avoid possible interactions with other libraries and for consistency if you need to potentially add other environment variables in the future, it would be useful to pick a consistent prefix (e.g., ``NI``) for NWB Inspector environment variables, e.g,  ``NI_SKIP_NETWORK_TESTS``

Reply:
``NWBI`` seems fine. I would say "first come first serve" in this case. If we ever add anything else that may want to use ``NWBI`` as a prefix then we'll have to deal with that then.